tests/system/action/committee/test_json_upload.py

```diff
@@ -0,0 +1,298 @@
+from openslides_backend.action.mixins.import_mixins import ImportState
```

Comment:
Besides the mentioned problems above, I missed tests for:
-forwarding_to_committees, committee_managers, organization_tags: adding and removing of related tags, empty fields in csv-file don't put in payload and don't change anything

Reply:
Please give a example description of such an add and removing test. I don't understand this this way.

Reply:
Example for remove see below on test test_json_upload_forward_to_committees.

Change the test, where a committee is created by forwarding only (=> warning) and create a new one, where the forwarding points to another committee to be created in the same action.
Would be nice for both committees: A->B and B->A

Reply:
I add a circular test, in which the circle is created thru the import.